test/transition.js

```diff
@@ -3,20 +3,15 @@ var transition = require('../src/transition')
 var props = transition.properties;
 
 describe('Transition helpers', () => {
-
-  //beforeEach(()=>{
-  //   document.body.innerHTML = window.__html__['test/fixtures/event.html']
-  // })
-
   it('should parse duration from node property', () => {
     var el = document.createElement('div');
 
-    el.style[props.duration] = '1.4s'
+    el.style[props.duration] = '2s'
 
-    expect(transition.end._parseDuration(el)).to.equal(1400)
+    expect(transition.end._parseDuration(el)).to.equal(2000)
```

Comment:
> FAILED TESTS:
  Query helpers
    Offset
  Transition helpers
    ✖ should parse duration from node property
      Chrome 53.0.2785 (Mac OS X 10.12.2)
    Error: expected 1000 to equal 1400
        at Assertion.assert (webpack:///~/expect.js/index.js:96:0 <- test.js:2805:14)
        at Assertion.be.Assertion.equal (webpack:///~/expect.js/index.js:216:0 <- test.js:2925:11)
        at Context.<anonymous> (webpack:///test/transition.js:13:49 <- test.js:19252:51)